lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -106,7 +106,7 @@ def check_file_request(file, klass, validate_all = true, lines_to_validate = 0,
         }
 
         streamed_ndjson_get(file['url'], headers) do |response, resource|
-          assert response.header['Content-Type'] == 'application/fhir+ndjson', "Content type must be 'application/fhir+ndjson' but is '#{response.header['Content-type']}'"
+          assert response.header['Content-Type'].include?('application/fhir+ndjson'), "Content type must have 'application/fhir+ndjson' but found '#{response.header['Content-type']}'"
```

Comment:
It doesn't look like we've verified that the `Content-Type` header is present before this, so I think we need a nil-safe check:
```suggestion
          assert response.header['Content-Type']&.include?('application/fhir+ndjson'), "Content type must have 'application/fhir+ndjson' but found '#{response.header['Content-type']}'"
```